addons/web/static/src/scss/report.scss

```diff
@@ -59,3 +59,7 @@ div#total {
         }
     }
 }
+
+.container.o_in_iframe {
+    max-width: 100%;
```

Comment:
Feels strange ... :)

Reply:
Well I suppose I can remove class `.container` and add the padding on the `.o_in_iframe` instead?

Reply:
What's the exact goal of that rule ?

Reply:
To make the invoice inside the iframe take the full width of the iframe. There are too much white borders on the sides otherwise depending on the width of the iframe.

Reply:
If you cannot use `container-fluid` then ok.

Reply:
So the best solution for you is to replace `container` by `container-fluid` when in inframe?

Reply:
If possible, yes

Reply:
Done